nvflare/app_common/widgets/intime_model_selector.py

```diff
@@ -16,20 +16,20 @@
 
 from nvflare.apis.dxo import DataKind, MetaKey, from_shareable
 from nvflare.apis.event_type import EventType
-from nvflare.apis.fl_component import FLComponent
 from nvflare.apis.fl_constant import FLContextKey, ReservedKey
 from nvflare.apis.fl_context import FLContext
 from nvflare.apis.shareable import Shareable
 from nvflare.app_common.app_constant import AppConstants
 from nvflare.app_common.app_event_type import AppEventType
+from nvflare.widgets.widget import Widget
 
 
-class IntimeModelSelectionHandler(FLComponent):
+class IntimeModelSelector(Widget):
```

Comment:
This name change will brake many examples. Can you add an alias to the old name to keep backward capability?

Reply:
thanks for the feedback, I've added the alias down there.